imola-cache.lib

```diff
@@ -0,0 +1,335 @@
+EESchema-LIBRARY Version 2.3
```

Comment:
we usually do not include "imola-cache.lib" into git since the assumption is that we have working paths and all the libraries are right? what we remove this with one of the following commits?